2022-01-14 17:17:45,070 INFO    MainThread:609884 [wandb_setup.py:_flush():71] setting env: {}
2022-01-14 17:17:45,070 INFO    MainThread:609884 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 17:17:45,070 INFO    MainThread:609884 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 17:17:45,070 INFO    MainThread:609884 [wandb_init.py:_log_setup():371] Logging user logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_171745-3ut7i6gp/logs/debug.log
2022-01-14 17:17:45,071 INFO    MainThread:609884 [wandb_init.py:_log_setup():372] Logging internal logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_171745-3ut7i6gp/logs/debug-internal.log
2022-01-14 17:17:45,071 INFO    MainThread:609884 [wandb_init.py:init():404] calling init triggers
2022-01-14 17:17:45,071 INFO    MainThread:609884 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-14 17:17:45,071 INFO    MainThread:609884 [wandb_init.py:init():460] starting backend
2022-01-14 17:17:45,071 INFO    MainThread:609884 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-14 17:17:45,080 INFO    MainThread:609884 [backend.py:ensure_launched():216] starting backend process...
2022-01-14 17:17:45,084 INFO    MainThread:609884 [backend.py:ensure_launched():221] started backend process with pid: 609942
2022-01-14 17:17:45,085 INFO    MainThread:609884 [wandb_init.py:init():469] backend started and connected
2022-01-14 17:17:45,087 INFO    MainThread:609884 [wandb_init.py:init():533] updated telemetry
2022-01-14 17:17:45,098 INFO    MainThread:609884 [wandb_init.py:init():563] communicating current version
2022-01-14 17:17:45,483 INFO    MainThread:609884 [wandb_init.py:init():568] got version response 
2022-01-14 17:17:45,483 INFO    MainThread:609884 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-14 17:17:45,644 INFO    MainThread:609884 [wandb_init.py:init():606] starting run threads in backend
2022-01-14 17:17:45,700 INFO    MainThread:609884 [wandb_run.py:_console_start():1810] atexit reg
2022-01-14 17:17:45,701 INFO    MainThread:609884 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-14 17:17:45,701 INFO    MainThread:609884 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-14 17:17:45,702 INFO    MainThread:609884 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-14 17:17:45,702 INFO    MainThread:609884 [wandb_init.py:init():633] run started, returning control to user process
2022-01-14 17:17:46,740 INFO    MainThread:609884 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-14 17:17:46,741 INFO    MainThread:609884 [wandb_run.py:_restore():1752] restore
2022-01-14 17:17:49,229 INFO    MainThread:609884 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 17:17:49,471 INFO    MainThread:609884 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 17:17:49,647 INFO    MainThread:609884 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 7983
}

2022-01-14 17:17:49,752 INFO    MainThread:609884 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 7983
}

2022-01-14 17:17:49,855 INFO    MainThread:609884 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2460
  total_bytes: 7983
}

2022-01-14 17:17:49,956 INFO    MainThread:609884 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7983
  total_bytes: 7983
}

2022-01-14 17:17:50,058 INFO    MainThread:609884 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7983
  total_bytes: 7983
}

2022-01-14 17:17:50,159 INFO    MainThread:609884 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7983
  total_bytes: 7983
}

2022-01-14 17:17:50,260 INFO    MainThread:609884 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7983
  total_bytes: 7983
}

2022-01-14 17:17:50,501 INFO    MainThread:609884 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7983
  total_bytes: 7983
}

2022-01-14 17:17:50,876 INFO    MainThread:609884 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7983
  total_bytes: 7983
}
local_info {
}

2022-01-14 17:17:51,942 INFO    MainThread:609884 [wandb_run.py:_append_files():2180] logging synced files
